integration/apps/hpl_mkl/hpl_mkl.py

```diff
@@ -44,8 +44,8 @@ class HplMklAppConf(hpl_netlib.HplNetlibAppConf):
     def name():
         return 'hpl_mkl'
 
-    def __init__(self, num_nodes, mach, perc_dram_per_node=0.9, cores_per_node=None):
-        super(HplMklAppConf, self).__init__(num_nodes, mach, perc_dram_per_node, cores_per_node)
+    def __init__(self, num_nodes, mach, frac_dram_per_node=0.9, cores_per_node=None):
```

Comment:
are these defaults used at all?  if not it might be better to remove them

Reply:
They are not used currently, they are there in case these functions are called in another context. I will remove them.